backend/fms_core/services/sample.py

```diff
@@ -340,13 +340,14 @@ def prepare_library(process: Process,
                 coordinates=coordinates_destination if coordinates_destination else "",
                 creation_date=execution_date,
                 volume=volume_destination if volume_destination is not None else volume_used,
-                depleted=False
+                depleted=False,
```

Comment:
Should we add a depleted column to the lib prep template in case there is an issue and the resulting lib is junk ... ? Maybe we can ask that during our next meeting with the lab.